csharp/extractor/Semmle.Extraction.CSharp.DependencyFetching/FrameworkPackageNames.cs

```diff
@@ -4,7 +4,9 @@ namespace Semmle.Extraction.CSharp.DependencyFetching
 {
     internal static class FrameworkPackageNames
     {
-        public static string LatestNetFrameworkReferenceAssemblies { get; } = "microsoft.netframework.referenceassemblies.net481";
+        public const string LatestNetFrameworkMoniker = "net481";
```

Comment:
Hehe, TIL - "moniker" :-)

Reply:
TIL that `net481` is not an allowed moniker to pass to `dotnet new --framework`. (If it was, we wouldn't have to do the below regex mathcing)